pkg/kube/client.go

```diff
@@ -507,6 +507,7 @@ func updateResource(c *Client, target *resource.Info, currentObj runtime.Object,
 			return nil
 		}
 		// send patch to server
+		c.Log("Patch %s %q in namespace %s", target.Name, kind, target.Namespace)
```

Comment:
c.Log("Patch %s %q in namespace %s", target.Name, kind, target.Namespace) --> c.Log("Patch %s %q in namespace %s", kind, target.Name, target.Namespace)

Reply:
oh, sorry. I will change it.